content/reference/apis/configuration_variables/secrets.md

```diff
@@ -5,8 +5,19 @@ weight: 2
 
 Once a secret is uploaded via [wrangler](/tooling/wrangler/secret) or in the UI, the string is exposed on the global namespace as type [`String`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String).
 
-Example where `SECRET` is set  and then set as a header `Auth` in a fetch.
+In the below example, we set the secret `SECRET` using `wrangler secret`, and then use it in our script as part of the `Authorization` header.
+
+Creating the secret using [`wrangler secret`](/tooling/wrangler/secrets):
+
+```bash
+wrangler secret create SECRET
+Enter the secret text you'd like assigned to the variable name on the script named my-worker-ENVIRONMENT_NAME: mysekret
+```
+
+Using the secret inside of your Workers script:
 
 ```javascript
-let headers = new Headers({ Auth: SECRET })
+let headers = new Headers({ Authorization: SECRET })
 ```
+
+TODO: copy about `wrangler secret` vs Cloudflare dashboard UI secrets
```

Comment:
Hmm this is tricky. This section `/reference/apis/..` should strictly contain reference material to the runtime's global scope.  The wrangler command example here is a how-to. And how secrets work in the UI versus wrangler is an explanation article. 
One idea to resolve this is to put the wrangler secret create example as a template page as a snippet and then either just link to the template page here or remove this page since it's really just saying secrets are available as strings. Then in the Copy about `wrangler secrete create` versus UI   place under `/about/secrets` if it's a lot of information which btw how are they different?

Reply:
ah, i see! i'm happy to do whatever you think is best here 👍 doing a snippet works for me, let me know if you want me to commit that!